docs/bug-bounty.md

```diff
@@ -123,7 +123,7 @@ The Bug Bounty Program is not open to any of the following persons (“Ineligibl
 
 We may modify the terms of this program or terminate this program at any time. We won’t apply any changes we make to these program terms retroactively.
 
-All conditions specified in the [general terms agreement of colony.io](https://xdai.colony.io/terms-and-services) apply to this program.
+All conditions specified in the [general terms agreement of colony.io] apply to this program.
```

Comment:
Don't we have a new one? Then this sentence doesn't really make much sense, does it?